systemtest/src/main/java/io/strimzi/systemtest/performance/PerformanceConstants.java

```diff
@@ -9,6 +9,11 @@
  */
 public interface PerformanceConstants {
 
+    // --------------------------------------------------------------------------------
+    // ------------------------------ COMMON OPERATOR -----------------------------------
+    // --------------------------------------------------------------------------------
+    String OPERATOR_OUT_RECONCILIATION_INTERVAL = "OUT: Reconciliation interval (ms)";
```

Comment:
It is a bit confusing when you name the constant like that :D but I thing you are keeping it everywhere else as well.